CHANGELOG.md

```diff
@@ -14,20 +14,23 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   You can however create and export seccomp-bpf for them.
 - `SeccompError::sysrawrc()` that queries the system's raw error code directly returned
   by `ScmpFilterAttr::ApiSysRawRc`.
-- `ScmpFilterContext::export_bpf_mem`
+- `const-syscall` support for loongarch64 and m68k
 
 ### Changed
 
 #### Non-breaking
 - `get_api_sysrawrc` and `set_api_sysrawrc` can now be used with any API level.
 - `get_ctl_optimize` and `set_ctl_optimize` can now be used with any API level.
 - Rust 2021 Edition
+- updated `const-syscall` syscall definitions, including definitions for
+  `atomic_barrier`, `atomic_cmpxchg_32` and `getpagesize`
```

Comment:
Can these be move to the the existing changelog in the `Added` section as follows?

```
- `futex_waitv`, `set_mempolicy_home_node`, atmoic_barrier`,
  `atmoic_cmpxchg_32` and `getpagesize` syscalls for the `const-syscall` feature
```